src/index.js

```diff
@@ -1,19 +1,28 @@
 import humane from 'wavded/humane-js';
 import {inject} from 'aurelia-framework';
 import {I18N} from 'aurelia-i18n';
+import {Optional} from 'aurelia-dependency-injection';
 
-@inject(humane, I18N)
+@inject(humane, new Optional(I18N))
 export class Notification {
-  constructor (humane, i18n) {
-    this.humane = humane;
+  constructor(_humane, i18n) {
+    this.humane = _humane;
     this.i18n   = i18n;
   }
 
-  success (message) {
-    this.humane.log(this.i18n.tr(message), {addnCls: 'success'});
+  log(message, options) {
+    if (this.i18n && this.i18n.i18next.isInitialized()) {
```

Comment:
Remove `this.i18n &&`; but keep `this.i18n.i18next.isInitialized()`. That way, we only use it if it's configured and it automatically works.